MC-39142: [Documentation] Update documentation for DMT timezone settings (#8349)
magento/devdocs: src/guides/v2.3/migration/migration-migrate-follow-up.md

After migration, Customer Segments must be resaved from the [Admin](https://glossary.magento.com/admin) Panel to get them up and running.

### Configure time zone

> The tool does not migrate Timezone settings and administrator should manually configure them after the migration at Magento 2 Admin panel > Stores > Configuration > Locale Options > Timezone.

Reviewer: ```suggestion
The tool does not migrate timezone settings, so you must manually configure the timezone after migration at **Stores** > **Configuration** > **Locale Options** > **Timezone**.
```

> By default, Magento stores time data in UTC-0 zone in the database and displays it according to current Timezone settings.

Reviewer: ```suggestion
By default, Magento stores time data in the UTC-0 zone in the database and displays it according to the current timezone settings.
```

> If time data already saved in the database in the zone different from UTC-0, developer or administrator need to convert the existing time to UTC-0 using the Data Migration Tool’s \Migration\Handler\Timezone handler.

Reviewer: ```suggestion
If time data has already been saved in the database in a zone other than UTC-0, you must convert the existing time to UTC-0 using the Data Migration Tool’s `\Migration\Handler\Timezone` handler.
```

> In the following example, the Magento 1 has been wrongly saving time in UTC-7 to the database, for example, due to faulty 3rd party extension. To convert the customer account creation time to UTC-0 upon migration properly, add the following rule to map-customer.xml:

Reviewer: ```suggestion
In the following example, Magento 1 has been incorrectly saving time in the UTC-7 zone in the database (for example, due to a faulty third-party extension). To properly convert the customer account creation time to the UTC-0 zone upon migration, add the following rule to the `map-customer.xml` file:
```
